src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
To be clear, let's just track the version of the GAPIC generated code as `gapic/{version}`.